### PR TITLE
chore: add a known wallet

### PR DESCRIPTION
**Pull request by kalgoop**
## Types of changes

- [x] Other... Please describe: Update config with a known wallet

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

**Comment by dated** [comment]
since you're at it, can you replace `ARK` with `Ark` on the other wallets?

**Comment by kalgoop** [comment]
Oh sorry ! did Ark->ARK instead of ARK -> Ark

File: config/networks/mainnet.json [hunk 1]
```diff
@@ -24,11 +24,12 @@
...
-    "AZmQJ2P9xg5j6VPZWjcTzWDD4w7Qww2KGX": "ARK GitHub Bounty",
-    "ALgvTAoz5Vi9easHqBK6aEMKatHb4beCXm": "ARK Shield",
-    "AUDud8tvyVZa67p3QY7XPRUTjRGnWQQ9Xv": "ARK Team",
-    "AWkBFnqvCF4jhqPSdE2HBPJiwaf67tgfGR": "ARK Community Fund",
+    "AYCTHSZionfGoQsRnv5gECEuFWcZXS38gs": "Ark Bounty",
+    "AZmQJ2P9xg5j6VPZWjcTzWDD4w7Qww2KGX": "Ark GitHub Bounty",
+    "ALgvTAoz5Vi9easHqBK6aEMKatHb4beCXm": "Ark Shield",
+    "AUDud8tvyVZa67p3QY7XPRUTjRGnWQQ9Xv": "Ark Team",
+    "AWkBFnqvCF4jhqPSdE2HBPJiwaf67tgfGR": "Ark Community Fund",
+    "ANkHGk5uZqNrKFNY5jtd4A88zzFR3LnJbe": "Ark Hot Wallet",
```
Comment (j-a-m-l) [inline config/networks/mainnet.json]: @kalgoop would you mind to explain what is "Ark Hot Wallet"?

Comment (boldninja) [reply config/networks/mainnet.json]: Its our wallet for expenses Juan, its marked same on explorer.

Comment (j-a-m-l) [reply config/networks/mainnet.json]: OK, thanks

**Pull request merged**